src/selectedtests/datasource/cli.py

```diff
@@ -38,4 +37,4 @@ def create_indexes(ctx):
 
 def main():
     """Entry point for setting up selected-tests db indexes."""
-    return cli(obj={})
+    return cli(obj={}, auto_envvar_prefix="SELECTED_TESTS")
```

Comment:
This is a cool click trick. click can automatically fill in arguments [based on environment variables ](https://click.palletsprojects.com/en/7.x/options/#values-from-environment-variables). This means the user can either specify an env var or specify the command line flag.

Reply:
Nice 👍